donor-portal/src/types/AidPackage.ts

```diff
@@ -0,0 +1,32 @@
+import {AidPackageItem} from "./DonorAidPackageOrderItem";
+
+export interface AidPackage {
+  packageID: number;
+  name: string;
+  description: string;
+  status: AidPackage.Status;
+  aidPackageItems: AidPackageItem[];
+}
+
+export interface DonorAidPackage {
+  packageId: number;
+  description: string;
+  name: string;
+  status:AidPackage.Status;
+  supplierID: number;
+  orderItems: AidPackageItem[];
+  totalAmount: number;
+  pledgedPercentage: number;
+}
```

Comment:
We don't need this interface, right? I think it was created before the API is done, and should be removed from the admin portal as well.

```suggestion
```